app/src/main/java/nl/eduvpn/app/entity/VpnConfig.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * This file is part of eduVPN.
+ *
+ * eduVPN is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * eduVPN is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with eduVPN.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package nl.eduvpn.app.entity
+
+import com.wireguard.config.Config
+import de.blinkt.openvpn.VpnProfile
+
+sealed class VpnConfig
```

Comment:
This class looks exactly the same as your CurrentVpn class. Maybe you could reuse one of them?

Reply:
I removed VpnConfig in https://github.com/eduvpn/android/pull/318/commits/e16a0b06986b43613fc9d8b54184350ef0d909d8. Not sure if this is better.